lib/mapper.js

```diff
@@ -66,6 +66,12 @@ module.exports = function mapper (routePlan) {
         route.autoreply = false
       }
 
+      // If there are middleware then set it
```

Comment:
Above this section, if `routeSet.middleware` has been provided, attach it to `defaultRoute` similar to how it is set below.